openqaoa/openqaoa-core/algorithms/baseworkflow.py

```diff
@@ -0,0 +1,464 @@
+#   Copyright 2022 Entropica Labs
```

Comment:
shouldn't we call this one "base_workflow.py", since the other files are called "(r)qaoa_workflow.py"?

Reply:
`baseworkflow.py` keeps inline with the naming conventions for `basebackend.py` rather than the more specific workflow file names that depend on algorithms like base `(r)qaoa_workflow.py`